Countries Info/Helpers/Constants.swift

```diff
@@ -0,0 +1,50 @@
+//
+//  Constants.swift
+//  Countries Info
+//
+//  Created by Oleh Stasiv on 07.12.2020.
+//
+
+import UIKit
+
+struct Constants {
```

Comment:
It's better to have your own constants struct for each place of use or if you want to use some general constants struct you should add a comment on what properties used for.